src/proxy/hdrs/URL.cc

```diff
@@ -551,8 +551,22 @@ URLImpl::set_path(HdrHeap *heap, const char *value, int length, bool copy_string
 void
 URLImpl::set_params(HdrHeap *heap, const char *value, int length, bool copy_string)
```

Comment:
Why do we still have set_params if we don't support params anymore?

Reply:
To keep compatibility as possible as we can, since we only deprecate them at this time.